### PR TITLE
Copy editing & formatting of new 5.3.2 documentation

### PR DESCRIPTION
**Pull request by tst-jcupp**

### CODE REVIEW

File: docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst [hunk 1]
```diff
@@ -840,3 +840,4 @@ Portable Globe and Server 5.2
    :class: inline
 .. |Broadcast globe icon| image:: ../../art/fusion/portable/broadcast_false.gif
    :class: inline line-sized
+
```
Comment (tst-hrajamoney) [inline docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst]: .. |Broadcast globe icon| image:: ../../art/fusion/portable/broadcast_false.gif
   :width: 38px
   :height: 31px

This icon needs to be fixed in portableUserGuideWinLinux.rst also.

Comment (tst-jcupp) [reply docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst]: committed fix.

File: docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_2_0.rst [hunk 2]
```diff
@@ -34,7 +34,7 @@ Release notes: Open GEE 5.2.0
       files are stored and downloaded separately from the rest of the
       Open GEE code.
 
-      **OpenGEE.org Web Site**. The Open GEE repository now includes the
+      **Open GEE.org Web Site**. The Open GEE repository now includes the
       source code for the Open GEE website, which can be viewed at
       `http://www.opengee.org <http://www.opengee.org>`_.
```
Comment (tst-hrajamoney) [inline docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_2_0.rst]: can be changed to opengee.org Website

File: README.md [hunk 3]
```diff
@@ -39,8 +52,18 @@ Earth Enterprise Fusion & Server currently run on 64-bit versions of the followi
 * Ubuntu 14.04 LTS
```
Comment (tst-nfarah) [inline README.md]: Let's remove Ubuntu 14 from list of supported OS by Open GEE 5.3.2 https://github.com/google/earthenterprise/blob/release_5.3.2/README.md

**Pull request merged**